apps/studio/components/interfaces/Home/ProjectList/EmptyStates.tsx

```diff
@@ -39,33 +39,36 @@ export const Header = () => {
 export const NoFilterResults = ({
   filterStatus,
   resetFilterStatus,
+  withinTableCell = false,
   className,
 }: {
   filterStatus: string[]
   resetFilterStatus?: () => void
+  withinTableCell?: boolean
```

Comment:
Matching `NoSearchResults`. I wonder if we should combine these two.

Reply:
makes sense i think! can extend NoSearchResults to handle custom text